pkg/codegen/go/doc.go

```diff
@@ -36,14 +36,9 @@ type DocLanguageHelper struct {
 var _ codegen.DocLanguageHelper = DocLanguageHelper{}
 
 // GetDocLinkForPulumiType returns the doc link for a Pulumi type.
-func (d DocLanguageHelper) GetDocLinkForPulumiType(pkg *schema.Package, typeName string) string {
-	moduleVersion := ""
-	if pkg.Version != nil {
-		if pkg.Version.Major > 1 {
-			moduleVersion = fmt.Sprintf("v%d/", pkg.Version.Major)
-		}
-	}
-	return fmt.Sprintf("https://pkg.go.dev/github.com/pulumi/pulumi/sdk/%sgo/pulumi?tab=doc#%s", moduleVersion, typeName)
+func (d DocLanguageHelper) GetDocLinkForPulumiType(_ *schema.Package, typeName string) string {
+	pulumiModuleVersion := "v3"
```

Comment:
Pull this out into a `const` at the top of the file, or if not, make it a `const` here?